app/utils/rentry/wrapper.py

```diff
@@ -118,11 +124,11 @@ def import_rentry_link(self):
             return
 
         try:
-            raw_url = rentry_link + "/raw"
-            response = urllib.request.urlopen(raw_url)
+            raw_url = f"{rentry_link}/raw"
+            response = requests.get(raw_url)
 
-            if response.getcode() == 200:
-                page_content = response.read().decode("utf-8")
```

Comment:
Why remove this? Particularly, `decode("utf-8")` could be useful

Reply:
i added it back now

Reply:
Noice